rss2email/error.py

```diff
@@ -43,13 +43,15 @@ def log(self):
 class TimeoutError (RSS2EmailError):
     def __init__(self, time_limited_function, message=None):
```

Comment:
`message=""`

then the check below can be left out

Reply:
The code as I've written it is correct. It would not be correct to append `': '` to `message` if it's an empty string.